test/smoke/FirefoxPrivateVPNUITest/FirefoxPrivateVPNUITest/Tests/DeviceTest.cs

```diff
@@ -30,6 +26,11 @@ public void TestInitialize()
         {
             this.browser = new BrowserSession();
             this.vpnClient = new FirefoxPrivateVPNSession();
+
+            // Resize browser to make vpn client and browser are not overlapped
```

Comment:
You should check whether the screen can accommodate for this.

Reply:
I did research on this. There is another way to do this: set the selected window to top level. But in this way, we need to create desktop session to loop through all windows and attach the window to a new session with top level capability. Sometimes our vpn client cannot be brought to front which caused a lot of failures in UI testing. We resize the browser window to ensure that browser window wont cover vpn client window.